custom_components/tuya_local/devices/solar_inverter2.yaml

```diff
@@ -0,0 +1,96 @@
+name: LIDL Inverter

```

Comment:
Device and entity naming should follow the HA naming convention of only capitalising the first word (abbreviations and proper names or trademarks are exceptions).
https://developers.home-assistant.io/blog/2022/07/10/entity_naming/